environment/src/main/java/com/sequenceiq/environment/environment/service/sdx/SdxService.java

```diff
@@ -67,4 +69,8 @@ public void stopByCrn(String crn) {
         }
     }
 
+    @Override
+    public Optional<String> getPaasSdxRemoteDataContext(String sdxCrn) {
```

Comment:
Why is this interface implemented if it is basically an error if someone calls it?

Reply:
sdx-connector used by environment and core. core has the implementation and sdx-connector needs the interface implemented to inject it into PaasService.

Reply:
otherwise spring won't be able to instantiate PaasSdxService, thus PlatformAwareSdxConnector

Reply:
and I prefer interface (and not abstract class) in order to force developer think about it when he/she tries to add sdx-connector to an existing module